### DISCUSSION TITLE
GD00Z-7 Garage Opener Doesn't Report State Automatically

### DISCUSSION THREAD

**Discussion by mostlychris** [zwave-js/zwave-js#6498] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

I upgraded to the latest version of zwave js ui a few weeks ago and then started having issues with my GD00Z-7 garage door controller. I restored from backup and all was well. Fast forward to today when I upgraded HA and this rquired an update to zwave ui. I am now having issues again.

The opener shows healthy in the UI and I can send commands to open and close the door. What is not working is the reporting of door state (open/closed) unless I manually refresh the state in the UI. When I do that, it reports out to HA and shows the correct status in HA and the Zwave UI.

If I enable polling on the barrier state via the UI general settings page, it grabs the state correctly with each poll. Obviously this is not a long term solution as polling every 10 seconds is hard on the network.

What did you expect to happen instead?
The barrier state to update as soon as the state changes, as it has done in the past.

Upgrade to the latest version of the Z-Wave JS UI.
Open or close the garage door.
Observe that the barrier state did not update in the UI or anywhere else.
Manually refresh the Barrier Operator V1 section of the node and observe the barrier state reflect the current state of the door as well as update HA entities.

Anything else we should know?

This worked in versions of Zwave JS UI prior to t

### Software versions

Z-Wave JS UI:  3.0.1

### Device information

Manufacturer: Nortek Security & Control LLC
Model name: GD00Z-7
Node ID: 066

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[z-ui_2023-11-06.log](https://github.com/zwave-js/node-zwave-js/files/13272377/z-ui_2023-11-06.log)

[a comment by an automated account is not shown]

**Reply by AlCalzone** [reply]
The bot is right, you uploaded the wrong logfile.

**Reply by mostlychris** [reply]
Followed the directions. Let me try again.

**Comment by mostlychris** [comment]
[zwavejs_2023-11-07.log](https://github.com/zwave-js/node-zwave-js/files/13282707/zwavejs_2023-11-07.log)

**Comment by mostlychris** [comment]
It was a Monday. I didn't scroll down to the driver log instructions. Doh! I've added the correct log now.

**Comment by mostlychris** [comment]
Has there been any thoughts on this? Polling the device every 10 seconds is causing issues in my network and potentially draining some of my battery devices.

**Reply by AlCalzone** [reply]
Sorry, I must have missed that you posted the log. According to the manual, the device should have one association group which should contain the node ID of the controller. Please check in the groups tab that this is the case. If not, add it.

I suspect there was an issue during the original interview, which prevented it from being added automatically.

**Comment by mostlychris** [comment]
After all this, power cycling the device seems to have fixed it. I also removed the group endpoint and re-added it. Might have taken both efforts. Thanks @AlCalzone for your help!
